README.md

```diff
@@ -467,7 +467,9 @@ const index = pc.index('test-index');
 await index.fetch(['1']);
 ```
 
-The first argument is the name of the index you are targeting. There's an optional second argument for providing an index host override for all index operations.
+The first argument is the name of the index you are targeting. There's an optional second argument for providing an 
+index host override. Providing this second argument allows you to bypass the SDK's default behavior of resolving 
+your index host via the provided index name. You can find your index host in the [Pinecone console](https://app.pinecone.io).
```

Comment:
I'd also add something about using `describeIndex` or `listIndexes`:
```suggestion
your index host via the provided index name. You can find your index host in the [Pinecone console](https://app.pinecone.io), or by using `describeIndex` or `listIndexes`.
```
That feels clunky, but however you think it should be worded.